payment/payment_test.go

```diff
@@ -72,3 +76,18 @@ func TestP2WSH(t *testing.T) {
 		t.Errorf("TestSegwitAddress: error when encoding segwit")
 	}
 }
+
+func TestFromPublicKeys(t *testing.T) {
+	_, publicKey1 := btcec.PrivKeyFromBytes(btcec.S256(), privateKeyBytes1)
+	_, publicKey2 := btcec.PrivKeyFromBytes(btcec.S256(), privateKeyBytes2)
+
+	p2ms, err := payment.FromPublicKeys([]*btcec.PublicKey{publicKey1, publicKey2}, 1, &network.Regtest)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if hex.EncodeToString(p2ms.Script) != "5121036f5646ed688b9279369da0a4ad78953ae7e6d300436ca8a3264360efe38236e321023c61f59e9a3a3eb01c3ed0cf967ad217153944bcf2498a8fd6e70b27c7ab6ee652ae" {
+		t.Error("hax value of p2ms script not as expected")
```

Comment:
Typo
```suggestion
		t.Error("hex value of p2ms script not as expected")
```

I would also test both the ScriptHash and WitnessScriptHash addresses for this payment

Reply:
the resulting addresses should be:
* ScriptHash: `XJkohBHRMT8JUknSqCH7aJP9gAuAe9eNLY`
* WitnessScriptHash: `ert1q484pt3gqgthcxa35nl4t6utpd0uf7tkm240hlxe6k4newkydwcqs5sjc4c`